django_extensions/management/modelviz.py

```diff
@@ -121,9 +123,10 @@ def generate_graph_data(self):
                             relation['needs_node'] = False
 
     def get_graph_data(self, as_json=False):
-        now = datetime.datetime.now()
+        date = datetime.fromtimestamp(0) if self.deterministic else datetime.now()
```

Comment:
Can we change the variable name of `date` to `create_at` here ?

(When I am looking at the `date.strftime` below, I find myself questioning if this is the Python builtin date class aka `from datetime import date` or something else... So I think that would make it far more clear to read the code)